springfox-core/src/main/java/springfox/documentation/builders/OperationBuilder.java

```diff
@@ -222,10 +225,21 @@ public OperationBuilder tags(Set<String> tags) {
     this.tags.addAll(nullToEmptySet(tags));
     return this;
   }
+  
+  /**
+   * Updates the operation extensions
+   *
+   * @param extensions - operation extensions
+   * @return this
+   */
+  public OperationBuilder extensions(Map<String, Object> extensions) {
+    this.vendorExtensions = defaultIfAbsent(extensions, this.vendorExtensions);
```

Comment:
I would think this should be a merge operation and not a replace operation. Think of the situation where you'd have multiple plugins contributing to different extensions. The second plugins updates shouldn't override the first ones changes.